crypto/fipsmodule/bn/internal.h

```diff
@@ -185,8 +185,13 @@ void bn_mul_mont(BN_ULONG *rp, const BN_ULONG *ap, const BN_ULONG *bp,
 
 static inline void bn_umult_lohi(BN_ULONG *low_out, BN_ULONG *high_out,
                                  BN_ULONG a, BN_ULONG b) {
-#if defined(OPENSSL_X86_64) && defined(_MSC_VER) && !defined(__clang__)
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
This set of conditions breaks compilation on x86 under MSVC 🤦‍♂️ I'll revert this file for now since we're using Clang for compiling under Windows ARM64 anyway.